EIPS/eip-7892.md

```diff
@@ -59,37 +71,107 @@ To facilitate these changes on the execution layer, the `blobSchedule` object sp
     "max": 9,
     "baseFeeUpdateFraction": 5007716
   },
-  "1740693335": {
+  "12000000": {
     "target": 24,
     "max": 48,
     "baseFeeUpdateFraction": 5007716
   },
-  "1743285335": {
+  "12300000": {
     "target": 36,
     "max": 56,
     "baseFeeUpdateFraction": 5007716
   }
 }
 ```
 
-On the consensus layer, a new parameter is added to the configuration:
+### Consensus layer configuration
 
-```
+A new `BLOB_SCHEDULE` field is added to consensus layer configuration, containing a sequence of entries representing blob parameter changes **after** `ELECTRA_FORK_EPOCH`. There exists one entry per fork that changes blob parameters, whether it is a regular or a Blob-Parameter-Only fork.
+
+```yaml
 BLOB_SCHEDULE:
-  - EPOCH: 348618
+  - EPOCH: 400000     ## A future anonymous BPO fork
     MAX_BLOBS_PER_BLOCK: 24
-  - EPOCH: 355368
+  - EPOCH: 420000     ## A future anonymous BPO fork
     MAX_BLOBS_PER_BLOCK: 56
+  - EPOCH: 440000     ## GLOAS_FORK_EPOCH; a future named fork introducing blob parameter changes
+    MAX_BLOBS_PER_BLOCK: 72
 ```
 
 The parameters and schedules above are purely illustrative. Actual values and schedules are beyond the scope of this specification.
 
-### Requirements
+**Requirements:**
+
+- Execution and consensus clients **MUST** share consistent BPO fork schedules.
+- The slot number in the EL's `blobSchedule` **MUST** align with the start of the epoch specified in the consensus layer configuration.
+- The `max` field in the EL's `blobSchedule` **MUST** equal the `MAX_BLOBS_PER_BLOCK` value in the consensus layer configuration.
+
+### Modified `compute_fork_digest`
+
+The `compute_fork_digest` helper is updated to account for BPO forks:
+
+```python
+# These basic types already exist. Copied here for reference.
+Epoch = int
+Version = bytes
+Root = bytes
+ForkDigest = bytes
```

Comment:
I think we should remove these type aliases as they are a bit misleading. In the consensus specs repo, we use [remerkleable](https://pypi.org/project/remerkleable/) for types.

* Epoch is a uint64: https://jtraglia.github.io/eth-spec-viewer/#custom_types-Epoch
* Version is Bytes4: https://jtraglia.github.io/eth-spec-viewer/#custom_types-Version
* Root is Bytes32: https://jtraglia.github.io/eth-spec-viewer/#custom_types-Root
* ForkDigest is Bytes4: https://jtraglia.github.io/eth-spec-viewer/#custom_types-ForkDigest

Reply:
Sure, we can do that, if that's the convention. My goal was to provide sufficient context to make the EIP relatively self-contained. Otherwise we're requiring the reader to have more context than needed.